HISTORY.md

```diff
@@ -11,6 +11,15 @@ The third number is for emergencies when we need to start branches for older rel
 
 Our backwards-compatibility policy can be found [here](https://github.com/python-attrs/cattrs/blob/main/.github/SECURITY.md).
 
+## 25.2.0 (2025-06-24)
+
+- **Potentially breaking**: {class}`cattrs.Converter` now accepts a `use_alias` parameters. 
```

Comment:
> Looks good, don't forget the changelog entry.

Done. Thereby noticed that also the change to `make_dict_unstructure_fn_from_attrs` was still missing, which I added in another commit. But unclear if also `cattrs.gen.typeddicts.make_dict_structure_fn` requires a change ❓

Reply:
Ok, removed it, and also the release date (I guess you'll fill that in once you trigger the release?). Further, I adjusted the style to of the bullet point to match the other entries.